var/spack/repos/builtin/packages/ip/package.py

```diff
@@ -32,8 +33,8 @@ class Ip(CMakePackage):
         preferred=True,
     )
 
-    depends_on("c", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+    depends_on("c", type="build")  # should be test
```

Comment:
@AlexanderRichert-NOAA do you mean that this dependency on a C compiler should only be for testing?

```suggestion
    depends_on("c", type="build", when="+tests")
```

Reply:
Basically yeah, but if it's going to impact the spec anyway, I'd just as soon leave the 'c' dependency and wait for a 'test' option for the language build types (at least my understanding is that there isn't one).